### ISSUE TITLE
Can't load old CoNIC baseline pretrained weights 

### ISSUE THREAD

**Issue by lucasRemedios** [vqdang/hover_net#256]
Hi,
I am unable to load the baseline pretrained CoNIC weights from the CoNIC Challenge using the current pipeline for running inference on whole slide images. I downloaded these weights about a year ago.

It doesn't look like these pretrained weights are still available on the conic challenge page.

Are these weights (state dict) still supported with the current model platform?

**Comment by lbertelo01** [comment]
<img width="932" alt="image" src="https://github.com/vqdang/hover_net/assets/98480071/bece22f6-8b68-4d74-a87c-5ee919a120e7">

Do you have the same issue as me? Were you already able to solve the issue?

**Comment by simongraham** [comment]
Yes, the CoNIC HoVer-Net was trained slightly differently and you won't be able to load it out of the box.
